frontend/src/components/details/SelectVariableComponent.jsx

```diff
@@ -16,23 +16,76 @@ class SelectVariableComponent extends Component {
   */
     state = {
         selectedVariable: 'variable 1',
+        secondVariable: 'variable 2',
         variables: [
-            { id: '1', country: 'variable 1' },
-            { id: '2', country: 'variable 2' },
-            { id: '3', country: 'variable 3' }
+            { id: '1', name: 'variable 1' },
+            { id: '2', name: 'variable 2' },
+            { id: '3', name: 'variable 3' }
         ]
     };
 
     handleSubmit = () => {
         alert('The choosen variable is: ' + this.state.selectedVariable);
     };
 
-    onDropDownItemSelectedHandler = (country) => {
-        const selectedVariable = country;
+    onDropDownItemSelectedHandler = (name) => {
+        const selectedVariable = name;
         this.setState({ selectedVariable });
     };
 
+    onSecondDropDownSelectedHandler = (name) => {
+        const secondVariable = name;
+        this.setState({ secondVariable });
+    };
```

Comment:
What is this for? If It belongs to the compared model, then we should better differentiate between the two variables. Maybe Divide the area where the variables can be selected also into two columns?

Reply:
I know what you are saying and it is true. that way looks better. 
but where should we put the commit button? 
but please don't hesitate to change the UI later. I will also change the UI of your components.

Reply:
> but where should we put the commit button?

mhmm that's a good question. I will think about it, maybe we could put it in the middle? between the variables?

Reply:
It wouldn't look good that way, I think